drivers/net/ethernet/aquantia/atlantic/hw_atl2/hw_atl2_utils_fw.c

```diff
@@ -462,6 +462,44 @@ static int aq_a2_fw_get_mac_temp(struct aq_hw_s *self, int *temp)
 	return aq_a2_fw_get_phy_temp(self, temp);
 }
 
+static int aq_a2_fw_set_wol_params(struct aq_hw_s *self, const u8 *mac, u32 wol)
+{
+	struct link_control_s link_control;
+	struct mac_address_aligned_s mac_address;
+	struct wake_on_lan_s wake_on_lan;
```

Comment:
netdev is strict about reverse x-mas trees.

https://docs.kernel.org/process/maintainer-netdev.html#tl-dr

```suggestion
	struct mac_address_aligned_s mac_address;
	struct wake_on_lan_s wake_on_lan;
	struct link_control_s link_control;
```